### ISSUE TITLE
Unable to build Docker image

### ISSUE THREAD

**Issue by karelmaxa** [WrenSecurity/wrenidm#157]
I can't build the Wren:IDM docker image on my local environment (os: ubuntu, arch: arm64, docker: 24.0.6).

Build fails when running tests for `openidm-ui-admin` module:

```
126.8 [INFO] --- frontend-maven-plugin:1.12.0:grunt (grunt) @ openidm-ui-admin ---
126.8 [INFO] Running 'grunt build:dev --target-version=6.2.0-SNAPSHOT' in /project/openidm-ui/openidm-ui-admin
127.5 [INFO] Running "notify_hooks" task
127.5 [INFO] 
127.5 [INFO] Running "copy:libs" (copy) task
127.6 [INFO] Copied 28 files
127.6 [INFO] 
127.6 [INFO] Running "copy:compose" (copy) task
127.9 [INFO] Created 179 directories, copied 718 files
127.9 [INFO] 
127.9 [INFO] Running "eslint:lint" (eslint) task
131.4 [INFO] 
131.4 [INFO] Warnings summary
131.4 [INFO]   one-var  6
131.4 [INFO] 
131.4 [INFO] Running "babel:source" (babel) task
136.2 [INFO] 
136.2 [INFO] Running "less:dev" (less) task
136.5 [INFO] >> 2 stylesheets created.
136.5 [INFO] 
136.5 [INFO] Running "copy:compiled" (copy) task
136.8 [INFO] Created 163 directories, copied 350 files
136.8 [INFO] 
136.8 [INFO] Running "copy:compiledMain" (copy) task
136.8 [INFO] Copied 1 file
136.8 [INFO] 
136.8 [INFO] Running "copy:transpiled" (copy) task
137.0 [INFO] Copied 299 files
137.0 [INFO] 
137.0 [INFO] Running "replace:buildNumber" (replace) task
137.0 [INFO] 
137.0 [INFO] Running "babel:test" (babel) task
137.7 [INFO] 
137.7 [INFO] Running "copy:test" (copy) task
137.8 [INFO] Created 56 directories, copied 2 files
137.8 [INFO] 
137.8 [INFO] Running "qunit:all" (qunit) task
168.0 [INFO] >> There was an error with headless chrome
168.0 [INFO] Fatal error: waiting for target failed: timeout 30000ms exceeded
168.0 [INFO] ------------------------------------------------------------------------
168.0 [INFO] Reactor Summary for Wren:IDM - Parent 6.2.0-SNAPSHOT:
168.0 [INFO] 
168.0 [INFO] Wren:IDM - Parent .................................. SUCCESS [  0.115 s]
...
168.0 [INFO] Wren:IDM - UI - Admin .............................. FAILURE [ 49.974 s]

```

**Comment by karelmaxa** [comment]
I'm able to build the project in a non-docker environment using Chromium installed via Puppeteer.

More detailed log:
```
Running "qunit:all" (qunit) task
  puppeteer:launcher Calling /usr/bin/chromium --no-sandbox --headless --allow-file-access-from-files --disable-dev-shm-usage --remote-debugging-port=0 --user-data-dir=/tmp/puppeteer_dev_chrome_profile-dzOsdY +0ms
  puppeteer:protocol:SEND ► {"method":"Target.setDiscoverTargets","params":{"discover":true},"id":1} +0ms
  puppeteer:protocol:RECV ◀ {"id":1,"result":{}} +0ms
>> There was an error with headless chrome
>> TimeoutError: waiting for target failed: timeout 30000ms exceeded
Fatal error: waiting for target failed: timeout 30000ms exceeded

```

**Comment by karelmaxa** [comment]
The issue was fixed in #158.
